content/kubeone/master/infrastructure/production_recommendations.en.md

```diff
@@ -4,4 +4,61 @@ date = 2020-04-01T12:00:00+02:00
 weight = 4
 +++
 
-TBD
\ No newline at end of file
+## AWS
+
+### ami_id
+
+{{% notice warning %}}
+It's very important to set this terraform variable to avoid future terraform attempts to recreate your control plane
+instance.
+{{% /notice %}}
+
+Can be set AFTER initial `terraform apply`, and you can find its initial discovered values in your terraform state by
+using:
+
+```bash
+terraform state show data.aws_ami.ami
+# data.aws_ami.ami:
+data "aws_ami" "ami" {
+    ...
+    id                    = "ami-00f6fb16625871821"
+    ...
+```
```

Comment:
Let's split this into two blocks.

Reply:
This is a command and its direct output, so why two blocks?

Reply:
It looks a little bit better. Also, you can't use the copy to clipboard functionality if both command and output are in the same block, as both would be copied. We already split command and its output in all other documents.